descope/management/common.py

```diff
@@ -0,0 +1,15 @@
+class MgmtV1:
```

Comment:
Why we didnt set these in the main common file?
Did you get into import loop? mmm i need to think if there is a way to overcome this..

Reply:
There's going to be a lot of endpoint that are strictly for management. They aren't common to everything and so I thought the separation made this clearer.